test/e2e/settings/fixedRowsBottom.spec.js

```diff
@@ -221,5 +221,28 @@ describe('settings', () => {
       expect(getBottomClone().height()).toBe(47);
       expect(getBottomLeftClone().height()).toBe(47);
     });
+
+    it('should not display double border when `window` is a scrollable container', () => {
```

Comment:
You changed the `cachedFixedRowsTop` logic but there is no test for that. Does this test cover all the changes?

Reply:
I can't find analogical glitch for the `fixedRowsTop` option. Thus, I wrote in the PR:

> I did similar change for `top` overlay although it probably doesn't change anything (I did it for objects consistency).

I decided to not add test which may be useless (passing on current `develop`).